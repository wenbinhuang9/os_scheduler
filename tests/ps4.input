3 5 5 100
1 0 10 100
5 5 5 100
4 2 5 100