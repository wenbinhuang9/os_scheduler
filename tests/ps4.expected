1 10 0 10
3 15 5 10
4 20 13 18
5 25 15 20
8 15